helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -51,7 +51,10 @@ data:
       {{- end }}
       {{- if index .Values "alfresco-elasticsearch-connector" "enabled" }}
       -Delasticsearch.createIndexIfNotExists=true
-      -Delasticsearch.host={{ index .Values "alfresco-elasticsearch-connector" "elasticsearch" "host" }}
+      -Delasticsearch.host={{ index .Values "alfresco-elasticsearch-connector" "elasticsearch" "host" | default .Values.global.elasticsearch.host }}
+      -Delasticsearch.port={{ index .Values "alfresco-elasticsearch-connector" "elasticsearch" "port" | default .Values.global.elasticsearch.port }}
+      -Delasticsearch.user={{ index .Values "alfresco-elasticsearch-connector" "elasticsearch" "user" | default .Values.global.elasticsearch.user }}
+      -Delasticsearch.password={{ index .Values "alfresco-elasticsearch-connector" "elasticsearch" "password" | default .Values.global.elasticsearch.password }}
```

Comment:
I'd rather have that part of a named tpl as well

Reply:
even if we are using it only in one place?

Reply:
not as useful of course but I would not consider it's a problem. It would provide a centralised place where we compute ES details.
Can also be done as part of another ticket up to you

Reply:
I think there is a problem with the approach of moving to a single tpl in the alfresco-content-common subchart: there are some values that make sense only in the main chart and that never exists in other subcharts or they are somewhere else. For `_helpers-activemq.tpl` worked because you are in the same chart so Values are consistent. With elasticsearch instead we have different charts and there is also a clash in Values: subchart have .Values.elasticsearch to provide overrides while the main chart at the same path has the section to configure the elasticsearch bitnami chart.

I think it's better to better discuss approach for the future, for now I suggest to move as is if we want this part of the next release.

Reply:
In the end I've created a new tpl but that lives in the parent chart so it doesn't imply can be used in every chart the depends on alfresco-content-common